crime-commons-mod-schemas/src/main/resources/schemas/ioj/common/iojAppeal.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "id": "iojAppeal.json",
+  "type": "object",
+  "title": "IoJ Appeal",
+  "description": "IoJ Appeal Data",
+  "properties": {
+    "receiptDate": {
```

Comment:
Would `receivedDate` make more sense here, just thinking that we use this identifier in other domains for when an application has been received or created etc.

Reply:
Interesting. I called it `receiptDate` after what it's referred to in MAAT. I'm not sure if there is a technical difference between date of receipt and received date - I'll ask our BA and get back to you on this.

Reply:
I guess actually thinking about it some more, there could be a slight difference. A receipt date might be more relevant if say the IoJ Appeal went for judicial review and a judge made a decision on the appeal which was then sent to the caseworkers. The received date (of that decision with the caseworker) might not necessarily be the same as the receipt date, or the date that the decision was made by the judge. In this case I think we'd want to capture the receipt date and not the received date, as it might impact the application.